index.js

```diff
@@ -9,6 +9,10 @@ const generateSeedPhrase = () => {
     return parseSeedPhrase(bip39.generateMnemonic())
 }
 
+const generateSeedPhraseWithEntropy = (entropy) => {
```

Comment:
@mattlockyer lets just make it optional parameter to generateSeedPhrase

Reply:
agreed